### PR TITLE
First pass review

### PR DESCRIPTION
**Pull request by phantas**
**Task:** https://trello.com/c/QsJLVOvJ

**Summary:**
As part of building a visualiser, I made a first pass review of the specification.

**Solution:**
This PR includes only minor changes, mostly typos and formatting. Bigger comments have been added to Trello for future work.

**UAT:**
_How can this be tested?_ N/A

**Notes:**
_Anything else the reviewer should be aware?_ Nope

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.idea/
+messaging.proto
+metadata.proto
+transport.proto
```
Comment (simonwo) [inline .gitignore]: How about `*.proto`?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  Copyright (C) 2017, The Personal Data Exchange Team, Crown Copyright (Government Digital Service).
+
+## 1. Introduction
+
+gov.uk Personal Data Exchange is a way to query existing personal data
+held by government. This distribution is the technical specification
+for how the system operates and a set of tools for working with the
+messages and protocols defined.
+
+There are a number of files supplied:
```
Comment (simonwo) [inline README.md]: The file list is already out-of-date... should we just remove this? We could replace it with a general description of what is in each folder, or do a per-file type thing.

**Review by simonwo — APPROVED** [review]
Lookin' good. Could we remove `FPR` from the commit message?

File: .gitignore (hunk 1 above)
Comment (phantas) [reply .gitignore]: Done. This was auto-generated :)

File: README.md (hunk 2 above)
Comment (phantas) [reply README.md]: Agree. I shuffled things around a bit and kept just the specification files and added a section about the examples directory. Better?

**Pull request closed without merging**